enterprise/backend/src/metabase_enterprise/data_editing/configure.clj

```diff
@@ -41,11 +41,10 @@
                    api/check-404)]
     {:title      (:name action)
      :parameters (for [param (:parameters action)]
-                   {:id          (case (:type action)
-                                   :query (:slug param)
-                                   :implicit (:id param))
+                   {:id          (:id param)
```

Comment:
@qnkhuat I never understood why we had this logic. If it was just to satisfying a hastily written outline of a test I wrote, then I apologize.